kalite/distributed/management/commands/initdconfig.py

```diff
@@ -5,10 +5,12 @@
 
 from django.conf import settings
 from django.core.management.base import BaseCommand
+import getpass
 
-script_template = """#! /bin/sh
+script_template = """
+#!/bin/sh
```

Comment:
This will create a script with a newline character as the first character, like this:
```

#!/bin/sh
...
```
Which I think is technically incorrect, as the `#!` should come first, right?